holoviews/tests/ui/bokeh/test_callback.py

```diff
@@ -268,39 +276,86 @@ def popup_form(name):
 
 @skip_popup
 @pytest.mark.usefixtures("bokeh_backend")
-def test_stream_popup_polygons_tap(serve_hv):
+@pytest.mark.parametrize("popup_position", [
+    "top_right", "top_left", "bottom_left", "bottom_right",
+    "right", "left", "top", "bottom"
+])
+def test_stream_popup_polygons_tap(serve_hv, popup_position):
     def popup_form(name):
-        return f"# {name}"
+        return "# selection"
 
     points = hv.Polygons([(0, 0), (0, 1), (1, 1), (1, 0)]).opts(tools=["tap"])
-    hv.streams.Tap(source=points, popup=popup_form("Tap"))
+    hv.streams.Tap(source=points, popup=popup_form("Tap"), popup_position=popup_position)
 
     page = serve_hv(points)
     hv_plot = page.locator('.bk-events')
     hv_plot.click()
     expect(hv_plot).to_have_count(1)
 
-    locator = page.locator(".markdown")
+    # Wait for popup to show
+    wait_until(lambda: expect(page.locator("#selection")).to_have_count(1), page)
```

Comment:
```suggestion
    expect(page.locator("#selection")).to_have_count(1)
```
`wait_until` and `expect` shouldn't be used simultaneously as they try to archive the same thing.